src/schemas/workflowSchemas.ts

```diff
@@ -123,3 +124,111 @@ export const workflowSchema = z.object({
 export const listWorkflowsResponseSchema = z
```

Comment:
The "listWorkflowsResponseSchema" is unused. I know that this is not your doing but removing it (and all the related code) would simplify this file. Boy scout rule.